util/perl/OpenSSL/Test.pm

```diff
@@ -1244,8 +1256,13 @@ sub __decorate_cmd {
 
     my $display_cmd = "$cmdstr$stdin$stdout$stderr";
 
-    $stderr=" 2> ".$null
-        unless $stderr || !$ENV{HARNESS_ACTIVE} || $ENV{HARNESS_VERBOSE};
+    # Make stdout/stderr redirection on non-verbosity dead code
+    # We're not yet sure this works properly everywhere, so we keep
+    # the code until we've experimented enough to be satisfied
+    if (0) {
```

Comment:
This should not remain - if(0) ... 

I see the comment - but it needs to go - you have git so you can go back to earlier versions.
Leaving "history" of something you might want to perhaps sometime in the future maybe go back to isn't the right move IMHO. Clean it out.

Reply:
Pardon me for being cautious, but I want to see how this fares on VMS first.  It's way too easy to "forget" and then wonder where the hell the previous code disappeared.

I will remove as soon as I've ensured his doesn't cause trouble there.  So the question is if you want to delay *this PR* on pure principle, or if you can find yourself waiting for a clean-up PR within a week or so?

I would normally just have this PR waiting, but this PR also solves a current Travis problem, so *this time* I can see reason to do that clean-up in a separate PR.

Reply:
So I rushed the experiment...  and it turns out that, for reasons I can't understand, some program output isn't caught by TAP::Parser on VMS.  So this code will be re-enabled, but only on VMS.